README.md

```diff
@@ -144,7 +144,8 @@ We have had to write our own `FindPackage` macros for most of the dependencies t
 
 ## Contributing
 
-We welcome new contributors! We will happily receive PRs for bug fixes or small changes. If you're contemplating something larger please get in touch first by opening a GitHub Issue describing the problem and how you propose to solve it.
+We welcome new contributors! We will happily receive PRs for bug fixes or small changes. If you're contemplating something larger please get in touch first by opening a GitHub Issue describing the problem and how you propose to solve it. 
+Once PR is oppened, make sure that `check-format` job is passing before checking `PR` as ready for review.
```

Comment:
Nice one! I would probably rephrase to get contributors to install and run `dotnet-format` locally before creating a PR. And mention of couple of popular IDEs that already support `.editorconfig` and will just do the right thing for them when they save the files.

Reply:
I was in the loop for some time about some words, but here is the result!